docs/docs/install.md

```diff
@@ -56,6 +56,12 @@ Download the appropriate release package from
 the [release page](https://github.com/go-gremlins/gremlins/releases/latest),
 extract the zip archive and copy the `.exe` file somewhere in your execution `PATH`.
 
+### :material-docker: Docker
+
```

Comment:
Here as well, you can add a sentence like: "You can also run Gremlins using the official Docker image: ..."